os/arch/arm/src/armv8-m/up_blocktask.c

```diff
@@ -155,8 +155,8 @@ void up_block_task(struct tcb_s *tcb, tstate_t task_state)
 
 	if (switch_needed) {
 #ifdef CONFIG_ARMV8M_TRUSTZONE
-		if (tcb->tz_context) {
-			TZ_StoreContext_S(tcb->tz_context);
+		if (rtcb->tz_context) {
```

Comment:
Is it correct to change `tcb` to `rtcb`?
Because 'tcb' is a tcb which will be blocked. So I think we should store its tz_context.

Reply:
Reverted it.